src/params.rs

```diff
@@ -50,46 +44,46 @@ pub const LITERALS: &[Num] = &[
     27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40,
 ];
 /// If not 0, include all numbers in 1..=MAX_LITERAL in addition to LITERALS.
-pub const MAX_LITERAL: Num = 0;
+pub const MAX_LITERAL: Num = 4;
```

Comment:
is this an intentional change?

Reply:
No, my bad, I didn't see that I changed it.